ntfs_fr.md

```diff
@@ -353,7 +353,8 @@ boarding_duration | entier | Optionnel | Durée nécessaire à l'embarquement en
 alighting_duration | entier | Optionnel | Durée nécessaire au débarquement en secondes (train, avion, ferry, etc.). Cette valeur est obligatoirement positive ou nulle.
 stop_id | chaine | Requis | Identifiant de l'arrêt physique de passage (cas général). Ce champ peut également référencer une "zone géographique" (stop de type 2) ou une commune (stop de type 3) dans le cas de TAD zonal.
 stop_sequence | entier | Requis | Ordre de passage de desserte dans la circulation
-stop_headsign | chaine | Optionnel |
+stop_headsign | chaine | Optionnel | Libellé qui doit être affiché au voyageur à la place du `trip_headsign` à cet arrêt.
+stop_short_name | chaine | Optionnel | Nom qui doit être affiché au voyageur à la place du `trip_short_name` à cet arrêt.
```

Comment:
`trip_short_name_at_stop` was suggested for this one in https://jira.kisio.org/browse/NAVP-942
I'd rather avoid the confusion with the stop's name.

Reply:
you're right ! thanks for spotting this :+1: